receiver/statsdreceiver/protocol/statsd_parser_test.go

```diff
@@ -1001,3 +1005,222 @@ func TestTimeNowFunc(t *testing.T) {
 	timeNow := timeNowFunc()
 	assert.NotNil(t, timeNow)
 }
+
+func TestStatsDParser_AggregateTimerWithHistogram(t *testing.T) {
+	timeNowFunc = func() time.Time {
+		return time.Unix(711, 0)
+	}
+	// It is easiest to validate data in tests such as this by limiting the
+	// histogram size to a small number and then setting the maximum range
+	// to test at scale 0, which is easy to reason about.  The tests use
+	// max size 10, so tests w/ a range of 2**10 appear below.
+
+	// Note: The structure here uses the mapping functions from
+	// OTel-Go sdk/metric/aggregator/exponential/mapping, which
+	// are the state of the OTel specification before the
+	// inclusivity adjustment here:
+	// https://github.com/open-telemetry/opentelemetry-specification/pull/2633,
+	// Tests that use exact powers-of-two will shift by one after
+	// this change which is the reason we avoid using exact powers
+	// of two (or anything near an exact power of two).
```

Comment:
Is this comment still accurate? We are not using OTel Go but `github.com/lightstep/go-expohisto/mapping/logarithm`, so I am not sure if the spec comment applies or not

Reply:
This was fixed, now removed.